harnessed_jobs/rtm_aliveness_exposure/v0/ccs_rtm_aliveness_exposure.py

```diff
@@ -0,0 +1,85 @@
+"""
+Exposure aliveness tests script, based on Homer's
+harnessed-jobs/T08/rebalive_exposure/ccseorebalive_exposure.py script.
+"""
+import sys
+import logging
+from org.lsst.ccs.scripting import *
+
+CCS.setThrowExceptions(True)
+
+logging.basicConfig(format="%(message)s",
+                    level=logging.INFO,
+                    stream=sys.stdout)
+logger = logging.getLogger()
+
+class CcsSubsystems(object):
+    def __init__(self, subsystems=None):
+        if subsystems is None:
+            subsystems = dict(ts8='ts8', rebps='ccs-rebps')
+        for key, value in subsystems.items():
+            self.__dict__[key] = CCS.attachSubsystem(value)
+
+def verify_rebs(ccs_sub, logger=logger):
+    rebs = ccs_sub.ts8.synchCommand(10, "getREBDeviceNames").getResult()
+    logger.info("%i REBs found:", len(rebs))
+    for reb in rebs:
+        logger.info("  %s", reb)
+
+def setup_sequencer(ccs_sub, sequence_file=sequence_file, nclears=10,
+                    logger=logger):
+    logger.info("Set up the sequencer and execute a zero-second exposure:")
+
+    command = "loadSequencer %s" % sequence_file
+    logger.info(command)
+    logger.info(ccs_sub.ts8.synchCommand(90, command).getResult())
+
+    command = "setSequencerStart Clear"
+    logger.info(command)
+    logger.info(ccs_sub.ts8.synchCommand(10, command).getResult())
+
+    command = "startSequencer"
+    for iclear in range(nclears):
+        logger.info("%i %s", iclear, command)
+        logger.info(ccs_sub.ts8.synchCommand(10, command).getResult())
+
```

Comment:
Good question.  I am replicating what is in the [BNL script](https://github.com/lsst-camera-dh/harnessed-jobs/blob/master/T08/rebalive_exposure/v0/ccseorebalive_exposure.py#L122).  I was hoping someone could clarify whether this is necessary.